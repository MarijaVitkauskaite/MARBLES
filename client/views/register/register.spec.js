import React from 'react';
import renderer from 'react-test-renderer';
import Register from './register';
// all for old version. needs update
test('renders correctly', () => {
    const tree = renderer.create(<Register />).toJSON();
    expect(tree).toMatchSnapshot();
});


