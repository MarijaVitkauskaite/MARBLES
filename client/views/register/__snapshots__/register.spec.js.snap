// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<RCTSafeAreaView
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#7197AC",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Image
    source={
      Object {
        "testUri": "../../../assets/Loose.png",
      }
    }
    style={
      Object {
        "bottom": "52.5%",
        "height": 300,
        "position": "absolute",
        "width": 200,
      }
    }
  />
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#D9D9D9",
        "borderRadius": 30,
        "bottom": "-20%",
        "height": 50,
        "marginBottom": 20,
        "opacity": 0.6,
        "width": "70%",
      }
    }
  >
    <TextInput
      autoCapitalize="none"
      onChangeText={[Function]}
      placeholder="EMAIL"
      placeholderTextColor="#353535"
      style={
        Object {
          "alignItems": "stretch",
          "flex": 1,
        }
      }
      testID="email-input-register"
    />
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#D9D9D9",
        "borderRadius": 30,
        "bottom": "-20%",
        "height": 50,
        "marginBottom": 20,
        "opacity": 0.6,
        "width": "70%",
      }
    }
  >
    <TextInput
      autoCapitalize="none"
      onChangeText={[Function]}
      placeholder="PASSWORD"
      placeholderTextColor="#353535"
      secureTextEntry={true}
      style={
        Object {
          "alignItems": "stretch",
          "flex": 1,
        }
      }
    />
  </View>
  <View
    accessible={true}
    collapsable={false}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "bottom": "-20%",
        "opacity": 1,
      }
    }
  >
    <Image
      source={
        Object {
          "testUri": "../../../assets/RegisterButton.png",
        }
      }
    />
  </View>
  <View
    accessible={true}
    collapsable={false}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "bottom": "-20%",
        "opacity": 1,
      }
    }
  >
    <Image
      source={
        Object {
          "testUri": "../../../assets/OrLogin.png",
        }
      }
      style={
        Object {
          "bottom": -120,
          "height": 50,
          "width": 80,
        }
      }
    />
  </View>
</RCTSafeAreaView>
`;
